<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="dry_checkBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>드라이</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="iron_checkBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>아이롱</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="outdoor_checkBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>190</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>잠바</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pent_checkBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>바지</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sweater_checkBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>스웨터</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pay_comboBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>510</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <item>
    <property name="text">
     <string>후불</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>선불</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>출고</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>입고/출고</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="long_checkBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>롱패딩</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ycloth_checkBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>250</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Y 셔츠</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cloth_checkBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>상의</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>185</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>후불/선불</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aregar_Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>640</y>
     <width>256</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="set_checkBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>한벌</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cote_checkBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>코트</string>
   </property>
  </widget>
  <widget class="QComboBox" name="inPutOutput">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <item>
    <property name="text">
     <string>입고</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>출고</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="repair_checkBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>185</width>
     <height>30</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>수선</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>작업</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="salario_lineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
